src/constructors/generalized_petersen.c

```diff
@@ -24,17 +24,17 @@
 /**
  * \function igraph_generalized_petersen
  * \brief Creates a Generalized Petersen graph.
- * 
```

Comment:
As above; please revert this file to its original content and then I'll fix up the whitespace in a separate commit.